snapcraft/templates/ros2/snap/snapcraft.yaml.j2

```diff
@@ -0,0 +1,37 @@
+# The name of the snap.
+# Use 'snapcraft register <name>' to register the name on the store.
+name: {{ name }}
+# Just for humans, typically '1.2+git' or '1.3.2'
+version: "0.0.1"
+summary: Single-line elevator pitch for your amazing snap # 79 char long summary
+description: |
+  This is {{ name }}'s description. You have a paragraph or two to tell the
+  most important story about your snap. Keep it under 100 words though,
+  so that it looks good in the snap store.
```

Comment:
It doesn't seem right to vary this from the `simple` profile, but I think I prefer keeping the comments on their own lines. Whatever we do, let's make it consistent between the two profiles.

@mr-cal Shall we enforce the 80-char rule here?

Reply:
As in, wrap the template at 80 characters? Or mention there is an 80 character limit in a comment?

I think wrapping at 80 characters is a good idea for readability.

Reply:
Wrap to 80 chars, yes.